File: download_map.js
const https = require('https');
const fs = require('fs');

const url = 'https://simplemaps.com/static/svg/co/co.svg';

https.get(url, { headers: { 'User-Agent': 'Mozilla/5.0' } }, (res) => {
    let data = '';
    res.on('data', (chunk) => { data += chunk; });
    res.on('end', () => {
        const match = data.match(/<svg[^>]*>([\s\S]*?)<\/svg>/i);
        if (match) {
            // Find the viewBox
            const vMatch = data.match(/viewBox="([^"]+)"/i);
            const viewBox = vMatch ? vMatch[1] : '0 0 1000 1000';

            fs.writeFileSync('assets/mapa-colombia.svg', data, 'utf8');
            console.log('Saved SVG to assets/mapa-colombia.svg');
            console.log('ViewBox:', viewBox);
            console.log(match[1].substring(0, 300));
        } else {
            console.log('No SVG match found. Data was:');
            console.log(data.substring(0, 300));
        }
    });
}).on('error', (err) => {
    console.log('Error:', err.message);
});

File: download_map.py
import urllib.request
import re

url = "https://upload.wikimedia.org/wikipedia/commons/e/ea/Blank_map_of_Colombia.svg"
req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
try:
    with urllib.request.urlopen(req) as response:
        svg_content = response.read().decode('utf-8')
        
    # Simplify presentation: set fill to #4b4e53 and stroke to white
    # Just output the raw first 1000 characters to see the structure
    print(svg_content[:1000])
    
    with open('assets/mapa-colombia.svg', 'w', encoding='utf-8') as f:
        f.write(svg_content)
    print("Saved to assets/mapa-colombia.svg")
except Exception as e:
    print("Error:", e)
